src/playground/workers/biomeWorker.ts

```diff
@@ -40,6 +40,8 @@ let configuration: undefined | Configuration;
 function getPathForFile(file: File): BiomePath {
 	return {
 		path: file.filename,
+		kind: ["Handleable"],
```

Comment:
As a quick fix, this is functional
```suggestion
		kind: 1 << 4,
```